src/problem/problems.jl

```diff
@@ -7,6 +7,20 @@ is the abstract type which defines a problem.
 """
 abstract type ProblemType end
 
+"""
+The structure 
+
+    EmptyProblem{S, XI, XT, XC, TC, T}
+
+encodes consistig in constructing the abstraction in X 
```

Comment:
Me no comprendo

Reply:
The goal is to construct the abstraction within a specified region, without addressing control application at this stage. While the name might not be the most precise, this approach allows the user to first focus on building the abstraction. Once that is complete, they can then move on to solving problems such as optimal control or safety verification.

Moreover, the UniformGrid solver is structured into sub-solvers, each dedicated to solving specific tasks like optimal control or safety analysis within the symbolic model. It also includes a solver specifically designed for computing the abstraction. To maintain consistency with this structure, I have introduced a dedicated problem formulation focused solely on abstraction construction.

Reply:
I understand better now, thanks. Can you update the docstring by fixing the typo like `consistig` an maybe copy-pasting part of your comment ?